### DISCUSSION TITLE
Pre filled relation select

### DISCUSSION THREAD

**Discussion by laravelbap** [filamentphp/filament#2812] [Help] [answered]
Hello,

When creating a related element using relation manager i want a pre-fill selected field with parent id (in my case property_id).
I have Property -> hasMany rooms

```
Forms\Components\BelongsToSelect::make('property_id')->relationship('property', 'name')->searchable(),
```
i couldn't find this in documentation. Is this possible?

**Comment by laravelbap** [comment]
I have solved this issue using this code (From another discussion)
...
Forms\Components\TextInput::make('exhibitor_name')->default( function (Livewire $livewire) {
    return $livewire->ownerRecord->name;
} );
...

**Comment by danharrin** [comment, marked as answer]
remove the field from the form, then the new record will be attached to the correct record anyway without extra work

**Reply by laravelbap** [reply]
Thanks. Your way is even better.
